src/locales/uk.ts

```diff
@@ -1377,7 +1377,7 @@ export default {
 	"pages.room.copy.lesson.message.copied": "Тему успішно скопійовано.",
 	"pages.room.copy.task.message.copied": "Завдання успішно скопійовано.",
 	"pages.room.itemDelete.text":
-		'Ви впевнені, що хочете видалити елемент " {itemTitle} "?',
+		"{itemTitle} {itemType} буде видалена. Ви впевнені, що хочете видалити?",
```

Comment:
```suggestion
		"{itemType} {itemTitle} буде видалена. Ви впевнені, що хочете видалити?",
```
I guess `itemType` and `itemTitle` need to switch places here in order to be correct.